### DISCUSSION TITLE
Is there a way to display the first word of a field?

### DISCUSSION THREAD

**Discussion by kbarkhurst** [statamic/cms#4358] [Q&A] [answered]
I've been reviewing the Statamic modifiers and wanted to make sure I wasn't overlooking a way to display the first word of a field.

In my case, I have a field that has a First Name and Last Name in one field. I was hoping to pull out the first name by grabbing just the first word in a field. Everything before the first &NBSP;

Any ideas on a way to do this? 

Alternatively, in my case, the slug is firstname-lastname too, so I even thought of being able to grab the first word within the slug (all letters before the first dash).

Thanks!

**Comment by jasonvarga** [comment, marked as answer]
```
{{ var explode=" " first="true" }}
```

I would have suggested `{{ var | explode:" " | first }}` but it doesn't look like a space works in explode there.

**Reply by kbarkhurst** [reply]
Works! Thanks!

**Comment by dgedwards13** [comment]
Another potential solution (If you know the last name is only one word) is the `regex_replace` modifier. Probably not as elegant as the above solution, but it's another tool in the toolbelt.

Input:
```
name: First Last
```

Antlers:
```
{{ title regex_replace='\\s+\\w+$|' }}
```
Output:
```
First
```

**Reply by kbarkhurst** [reply]
Thanks for sharing this alternative solution. I'm still learning regex, would you mind sharing a bit of what the code is doing? 
I believe \\s matches with whitespace character and \\w matches a word character but I'm unsure what +$| means.

**Reply by jasonvarga** [reply]
It searches for:

- `\s` a space
- `+` one or more (spaces)
- `\w` is a letter/number
- `+` again, one or more (letters/numbers)
- `$` end of the line

Then it replaces what it found with nothing, effectively chopping off the last word.
